plugins/terraform/terraform.go

```diff
@@ -26,6 +26,8 @@ func TerraformCLI() schema.Executable {
 				Optional: true,
 				NeedsAuth: needsauth.IfAny(
 					needsauth.ForCommand("refresh"),
+					needsauth.ForCommand("init"),
+					needsauth.Forcommand("state list"),
```

Comment:
`ForCommand` expects every arg to be passed separately.

```suggestion
					needsauth.Forcommand("state", "list"),
```